code-challenges.js

```diff
@@ -4,58 +4,106 @@
 
 // Pseudo coding is required.
 
-// --------------------INSTRUCTOR EXAMPLE: Create a function that takes in any two strings and returns the string with more characters. Use the two sets of test variables provided.
+// --------------------INSTRUCTOR EXAMPLE: Create a function that takes in any two strings and returns the string with more characters.
+// Use the two sets of test variables provided.
 
-// Pseudo code:
+// Pseudo code: 
+// create a function with a name myFruits
+// Arguments are two strings
+// Returns the string with more characters.
 
 // Set one:
 const fruit1 = "apple"
 const fruit2 = "banana"
-
+ 
+if (fruit1.length < fruit2.length) {
+    return fruit1
 // Set two:
 const fruit3 = "cherry"
 const fruit4 = "kiwi"
 
+ if (fruit3.length < fruit4.length ) {
+        return fruit4}
+}
+console.log(myFruits(fruit1, fruit2))
+console.log(myFruits(fruit3, fruit4))
 
 // --------------------1) Create a function that takes in any given number and determines if the number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below. Expected output: "42 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
-// Pseudo code:
+// Pseudo code: create function name it myTemp
+// argument is any number
+// returns if number is below boiling point
 
 const temp1 = 42
 const temp2 = 350
 const temp3 = 212
 
+const myStove = (temp) => {
+    if (temp < 212) {
+        return "Cannot boil water"
+    } else if (temp > 212 ) {
+        return "Can boil water"
+    }
+}
+console.log(myStove (350))
+
```

Comment:
Please remember to do a function call using each test variable. What will happen when 212 is used as an argument with this function?